### PR TITLE
Update  documentation jax.rst

### PR DESCRIPTION
**Pull request by ankit27kh**
`prng_key` does not exist anymore. `seed` seems to have taken its place

### CODE REVIEW

**Review by albi3ro — APPROVED** [review]
Thanks for catching this! Looks good to me 🚀

**Review by JerryChen97 — APPROVED** [review]
